src/external/bot-skeleton/services/tradeEngine/utils/interpreter.js

```diff
@@ -233,6 +233,9 @@ const Interpreter = () => {
     }
 
     function run(code) {
+        if (!interpreter || !(interpreter instanceof JSInterpreter)) {
+            interpreter = new JSInterpreter(code, initFunc);
+        }
```

Comment:
@maryia-matskevich-deriv no need to assign a new instance of the `JSInterpreter` it requires some additional properties which is assigned from top classes. Let's just put the bottom condition before calling the revert and it should be fine for now.

Reply:
done